Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16.19-bullseye-slim
+FROM node:20.2-bullseye-slim
```

Comment:
Have you ran this? This might be a breaking change

Reply:
No, I haven't. I've just used the node version recommend by @mohamedlajmileanix. 
It's the only version that solve the vulnerability.

Reply:
Fair enough, Lets make sure the actions runs successfully after the merge, otherwise we revert it